pkgs/development/compilers/gcc/12/default.nix

```diff
@@ -70,6 +70,10 @@ let majorVersion = "12";
         sha256 = "sha256-omclLslGi/2yCV4pNBMaIpPDMW3tcz/RXdupbNbeOHA=";
       })
       ++ optional langD ../libphobos.patch
+      ++ optionals (targetPlatform.libc == "serenitylibc") [
+        ./serenity-support.patch
+        ./serenity-libtool-enable-shared-library-support.patch
```

Comment:
Serenity's Ports directory seems to have copies of this same patch for just about every package using libtool, which worries me. Are we going to need to do that? Why not fix libtool/autoconf/whatever?

Reply:
In my limited testing the libtool patch doesn't seem to be needed anymore? 🤔 Not sure what might have changed in serenity/autoconf/libtool/etc in the last few months, but definitely welcome if it really has resolved the issue for good!